Makefile

```diff
@@ -136,7 +136,7 @@ draw-deps:
 	@goviz -i ./cmd/ag0 -d 2 | dot -Tpng -o dependency-graph.png
 
 clean:
-	rm -rf $(BUILDDIR)/ artifacts/
+	rm -rf $(BUILDDIR)/ build.linux/ artifacts/
```

Comment:
Having started my career in the age of 1000 UNIXes, I prefer `build.$(ARCH)` or something instead of `build.linux`